core/src/main/java/io/questdb/std/MmapCache.java

```diff
@@ -210,8 +228,9 @@ public long mremap(int fd, long mmapCacheKey, long address, long previousSize, l
      * Unmaps memory region, decrements reference count, and removes from cache if last reference.
      */
     public void unmap(long address, long len, int memoryTag) {
-        if (address == 0 || len <= 0) {
-            return;
+        if (address <= 0 || len <= 0) {
```

Comment:
nit: this change also creates inconsistency with `ff.close()` which doesn't throw when it's called on zero fd. This could lead to confusion. So, I wonder if we really need to prevent `ff.munmap(0)` calls. What kind of bug are we fixing with this?

Reply:
Yeah, to me it's worth it aligning the calls or we'll keep stumbling upon this behavioral difference.